src/components/KRadio/KRadio.vue

```diff
@@ -376,51 +370,83 @@ $kRadioDotSize: 6px;
 
       &.has-label, &.has-description {
         .card-content-wrapper {
+          height: auto;
```

Comment:
nit/question: why is this needed? Isn't it the default?

Reply:
This is needed, it overrides another rule just 3 lines above it